lib/Runtime/Debug/TTSupport.cpp

```diff
@@ -6,6 +6,14 @@
 
 #if ENABLE_TTD
 
+void TTDHardAbort(const char* msg)
```

Comment:
>TTDHardAbort [](start = 5, length = 12)

Can we rename this (TTD_fatal_error) to be inline with other fatal_error we have like JavascriptDispatch_OOM_fatal_error. The reason is watson analysis sometimes ignore some call stack when bucketing and we explicit include *fatal_error*